components/LoadingSpinner.tsx

```
@@ -0,0 +1,30 @@
+interface LoadingSpinnerProps {
+	className?: string;
+}
+
+export default function LoadingSpinner({ className }: LoadingSpinnerProps) {
+	return (
+		<svg
+			className={clsx("animate-spin", className)}
+			xmlns="http://www.w3.org/2000/svg"
+			fill="none"
+			viewBox="0 0 24 24"
+		>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)